tt_eager/tt_dnn/op_library/backward/backward_ops.hpp

```diff
@@ -44,17 +44,39 @@ std::vector<Tensor> addcmul_bw(
     float value,
     const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
 
-std::vector<Tensor> unary_mul_bw(
+std::vector<std::optional<Tensor>> unary_mul_bw(
     const Tensor& grad,
```

Comment:
As mentioned before, having unary_ or binary_ is just ugly.   Let's fix this in the migration to ttnn.